addons/spreadsheet/static/tests/global_filters/global_filters_model_test.js

```diff
@@ -2331,13 +2334,71 @@ QUnit.module("spreadsheet > Global filters model", {}, () => {
             id: "42",
             label: "test",
             type: "date",
-            defaultValue: {},
+            defaultValue: "",
             rangeType: "relative",
         };
         const result = await addGlobalFilter(model, filter);
         assert.ok(result.isSuccessful);
     });
 
+    QUnit.test("Cannot create a fixedPeriod date filter with a disabled value", async (assert) => {
+        const model = new OdooSpreadsheetModel();
```

Comment:
not sure it's bullet proof 
If I dispatch the following command:
```json
{
  "type": "ADD_GLOBAL_FILTER",
  "filter": {
    "id": "ee051dd6-0050-48a8-ac75-c41e72893aec",
    "type": "date",
    "label": "hj",
    "defaultValue": "this_quarter",
    "rangeType": "fixedPeriod",
    "disabledPeriods": ["month", "quarter"]
  },
  "pivot": {
    "1": {
      "chain": "date",
      "type": "date",
      "offset": 0
    }
  }
}
```
-> default value as quarter and reject quarters, the command is still accepted.

Reply:
I think the issue is that the default values of a fixedPeriod filter are not actually months name but `"this_month" | "this_quarter" | "this_year"`. The typing is wrong :/

Reply:
Yup you're right. I added this case int he allowDispatch, and changed the "automatic values" in the side panel so we wannot select a disabled value